tests/container/BUILD

```diff
@@ -320,6 +320,21 @@ container_push(
     tags = ["manual"],
 )
 
+container_image(
+    name = "alpine_with_custom_attr",
+    base = "@new_alpine_linux_amd64//image",
+    cmd = ["Hello World!"],
+    entrypoint = ["echo"],
+    files = [
+        "//testdata:bar",
+        "//testdata:foo",
+    ],
+    layers = [
+        "//testdata:env_layer",
```

Comment:
Can you add another layer here that has files? I want to ensure the pusher works when there's more than one layer with files to upload. Also, for this case we want to run a `container_pull` followed by `container_test` to ensure files from both layers were uploaded to the registry correctly.